include/material/bingham.h

```
@@ -0,0 +1,89 @@
+#ifndef MPM_MATERIAL_BINGHAM_H_
+#define MPM_MATERIAL_BINGHAM_H_
+
+#include <limits>
+
+#include "Eigen/Dense"
+
+#include "material.h"
+
+namespace mpm {
+
+//! Bingham class
+//! \brief Bingham fluid material model
+//! \details Bingham class stresses and strains
+//! \tparam Tdim Dimension
+template <unsigned Tdim>
+class Bingham : public Material<Tdim> {
+ public:
+  //! Define a vector of 6 dof
+  using Vector6d = Eigen::Matrix<double, 6, 1>;
+  //! Define a Matrix of 6 x 6
+  using Matrix6x6 = Eigen::Matrix<double, 6, 6>;
+
+  //! Constructor with id
+  Bingham(unsigned id) : Material<Tdim>(id){};
+
+  //! Destructor
+  virtual ~Bingham(){};
+
+  //! Delete copy constructor
+  Bingham(const Bingham&) = delete;
+
+  //! Delete assignement operator
+  Bingham& operator=(const Bingham&) = delete;
+
+  //! Read material properties
+  //! \param[in] materail_properties Material properties
+  void properties(const Json& material_properties );
+
+  //! Compute elastic tensor
+  //! \retval de_ Elastic tensor
+  Matrix6x6 elastic_tensor(){};
```

These need to throw a runtime error.